packages/jaeger-ui/src/utils/tracking/ga.tsx

```diff
@@ -56,7 +56,7 @@ const GA: IWebAnalyticsFunc = (config: Config, versionShort: string, versionLong
       msg = `jaeger/${msg}`;
     }
     msg = msg.slice(0, 149);
-    ReactGA.exception({ description: msg, fatal: false });
```

Comment:
does the new module not support a distinction between exceptions and other events?  The `send` in L155 specifies `hitType`, should this one also have `hitType=exception`?

Reply:
Ahh My bad this should be hitType=exception . Thank you.